### ISSUE TITLE
ViT-MAE softmax scores

### ISSUE THREAD

**Issue by LokeshBadisa** [jiaminchen-1031/transformerinterp#3]
The softmax values for teddy-polaroid camera image mentioned in the paper are logits rather than softmax scores for ViT-MAE. Is there anyway to revise the paper?
